### PR TITLE
PENG-1551 - Create charm-jobbergate-agent

### PR DESCRIPTION
**Pull request by fschuch**
Create a new charm for deploying the new `jobbergate-agent`. It should be almost identical to the cluster-agent 

Notice the `main` branch contains the code copied from cluster-agent, so this PR highlights just the changes to make it `jobbergate-agent`.

`Task`: https://app.clickup.com/t/18022949/PENG-1552

---

#### Peer Review
Please follow the upstream omnivector documentation concerning
[peer-review guidelines](https://github.com/omnivector-solutions/Documentation/blob/main/Contributing/pr_review_standards.md#peer-review).

### CODE REVIEW

File: src/charm.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 
 class ClusterAgentCharm(CharmBase):
```
Comment (jaimesouza) [inline src/charm.py:22]: I think this class should be renamed.

File: src/charm.py [hunk 2]
```diff
@@ -32,7 +32,7 @@ def __init__(self, *args):
         self.stored.set_default(config_available=False)
         self.stored.set_default(user_created=False)
 
-        self.cluster_agent_ops = ClusterAgentOps(self)
```
Comment (jaimesouza) [inline src/charm.py:35]: `ClusterAgentOps` should be` JobbergateAgentOps` also.

File: src/cluster_agent_ops.py [hunk 3]
```diff
@@ -7,57 +7,49 @@
 from shutil import copy2, rmtree
 from typing import Dict, Any
 
-from jinja2 import Environment, FileSystemLoader
-
-
 logger = logging.getLogger()
 
 
 class ClusterAgentOps:
```
Comment (jaimesouza) [inline src/cluster_agent_ops.py:13]: It should be` JobbergateAgentOps`.

**Review by jaimesouza — CHANGES_REQUESTED** [review]

File: src/cluster_agent_ops.py (hunk 3 above)
Comment (fschuch) [reply src/cluster_agent_ops.py]: solved in [ed479f6](https://github.com/omnivector-solutions/charm-jobbergate-agent/pull/1/commits/ed479f611c10625ed31eccf03fa680731684f725)

File: src/charm.py (hunk 2 above)
Comment (fschuch) [reply src/charm.py:35]: solved in [ed479f6](https://github.com/omnivector-solutions/charm-jobbergate-agent/pull/1/commits/ed479f611c10625ed31eccf03fa680731684f725)

**Pull request merged**